### PR TITLE
Use travis-ci for osx

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by jakirkham** [comment]
Why use Travis CI for this if Azure works?

**Comment by isuruf** [comment]
To reproduce https://github.com/conda-forge/libcxx-feedstock/issues/58

**Comment by jakirkham** [comment]
Ok so this is just for testing purposes. Thanks for clarifying.

**Comment by isuruf** [comment]
It's not just for testing. Azure image is too new for issues with older macOS to pop up. With travis-ci, we can test in an older macOS image.

**Comment by isuruf** [comment]
@timsnyder, @beckermr, using clang 9.0.0 to build libc++ fixes the original issue of missing `___cxa_uncaught_exceptions`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,12 +11,12 @@ source:
...
+  - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version }}/libcxxabi-{{ version }}.src.tar.xz  # [linux]
+    sha256: e71bac75a88c9dde455ad3f2a2b449bf745eafd41d2d8432253b2964e0ca14e1  # [linux]
+    folder: libcxxabi  # [linux]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win and vc<14]
   skip: true  # [win]
   skip: true  # [ppc64le or aarch64]
```
Comment (timsnyder) [inline recipe/meta.yaml:22]: Ugh. Github won't let me comment on the line I want to...  Can we re-enable `openturns` as a `test.downstreams`?  We removed it while trying to update to 10.0.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -57,7 +57,7 @@ test:
   # before releasing just to be sure.
   downstreams:   # [osx]
     - python-symengine  # [osx]
-    #- openturns         # [osx]
+    - openturns         # [osx]
```
Comment (timsnyder) [inline recipe/meta.yaml:60]: 👍

**Comment by timsnyder** [comment]
Using Travis to build libcxx with older sytem image seems like a better alternative to staying on Azure and potentially needing to bump the minimum supported osx APK.  Based on [gitter feedback](https://gitter.im/conda-forge/conda-forge.github.io?at=5e99bc8bd240da24337c2853) It seems like there are definitely users of conda-forge who still have OSX 10.10.  If we do want to bump the default minimum APK, we probably need to communicate it much better and give folks a chance to upgrade.

**Pull request merged**